src/plone/api/content.py

```diff
@@ -532,16 +532,21 @@ def get_view(name=None, context=None, request=None):
     # and get their names.
     available_view_names = [view[0] for view in available_views]
 
-    # Raise an error if the requested view is not available.
-    if name not in available_view_names:
-        raise InvalidParameterError(
-            "Cannot find a view with name '{name}'.\n"
-            "Available views are:\n"
-            "{views}".format(
-                name=name,
-                views="\n".join(sorted(available_view_names)),
-            ),
-        )
+
```

Comment:
The above logic to get available_view_names should also only be done if it is needed.

Reply:
Thank you for your feedback. I have made some changes accordingly. Hope it is alright.